documentation/source/development/engine-design/labeling.rst

```diff
@@ -1,5 +1,5 @@
-Labeling
-========
+GitHub Issue Labeling
+=====================
```

Comment:
Not sure if we should keep the list below. Maybe only the categories? (As it might be outdated very quick)

Reply:
It looks nice. I would keep it for now. We can adapt it once we change the issue labels, but I think they are very good currently.